obspy/clients/filesystem/tests/test_tsindex.py

```diff
@@ -0,0 +1,984 @@
+# -*- coding: utf-8 -*-
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+
+import unittest
+from obspy.core.compatibility import mock
+
+import obspy
+from obspy import UTCDateTime
+from obspy.core.compatibility import RegExTestCase
+from obspy.clients.filesystem.tsindex import Client, Indexer, \
+    TSIndexDatabaseHandler
+
+from collections import namedtuple
+
+import sqlalchemy as sa
+from sqlalchemy.orm import sessionmaker
+
+import uuid
+import os
+import re
+import requests
+import tempfile
```

Comment:
please arrange imports according to our standards

Reply:
I just pushed a fix for this. Please look at it to make sure it now meets the library standards, as I don't know where to find documentation on this.